src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,226 @@
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement
+                         = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            isComplite = statement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't delete a car by id " + id, e);
+        }
+        return isComplite;
+    }
+
+    private void removeRelation(Long carId) {
+        String query = "DELETE FROM cars_drivers WHERE car_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, carId);
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't remove car id: " + carId
+                    + " from cars_driver table.", e);
+        }
+    }
+
+    private Car parseCar(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        Manufacturer manufacturer = new Manufacturer();
+        car.setId(resultSet.getObject("cars.id", Long.class));
+        car.setModel(resultSet.getString("cars.model"));
+        manufacturer.setId(resultSet.getObject("manufacturers.id", Long.class));
+        manufacturer.setName(resultSet.getString("manufacturers.name"));
+        manufacturer.setCountry(resultSet.getString("manufacturers.country"));
+        car.setManufacturer(manufacturer);
+        return car;
+    }
+
+    private List<Driver> getCarDrivers(Long carId) {
+        String getAllDrivers = "SELECT id, name, license_number "
+                + "FROM drivers "
+                + "JOIN cars_drivers "
+                + "ON drivers.id = cars_drivers.driver_id "
+                + "WHERE cars_drivers.car_id = ? AND drivers.is_deleted = false;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(getAllDrivers)) {
+            statement.setLong(1, carId);
+            ResultSet resultSet = statement.executeQuery();
+            List<Driver> driverList = new ArrayList<>();
+            while (resultSet.next()) {
+                driverList.add(parseDriver(resultSet));
+            }
+            return driverList;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get all car driver by car id " + carId, e);
+        }
+    }
+
+    private Driver parseDriver(ResultSet resultSet) throws SQLException {
+        Driver driver = new Driver();
+        driver.setId(resultSet.getObject("id", Long.class));
+        driver.setName(resultSet.getNString("name"));
+        driver.setLicenseNumber(resultSet.getString("license_number"));
+        return driver;
+    }
+
+    public List<Car> getAllByDriver(Long driverId) {

```

Comment:
```suggestion
    @Override
    public List<Car> getAllByDriver(Long driverId) {

```

public methods should be above private